src/extension/background-script/connectors/nativecitadel.ts

```diff
@@ -58,40 +40,44 @@ export default class NativeCitadel extends NativeConnector {
     });
   }
 
-  async request<ResponseType = unknown>(
-    jwt: string,
-    url: string,
-    method?: "GET" | "POST" | "PUT" | "DELETE",
-    body?: unknown,
-    auth?: boolean
-  ): Promise<ResponseType> {
-    let authHeader = "";
-    if (jwt) authHeader = `JWT ${jwt}`;
+  async request<Type>(
+    method: string,
+    path: string,
+    args?: Record<string, unknown>
+  ): Promise<Type> {
+    let body;
     let headers: Record<string, string> = {};
+    path = this.config.url + (this.config.url.endsWith("/") ? "" : "/") + path;
```

Comment:
I think we should call this `url` because it is the full url and not only the path.